java/arcs/android/devtools/DevToolsStarter.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.devtools
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.annotation.VisibleForTesting
+import androidx.core.content.ContextCompat.startForegroundService
+import arcs.sdk.android.storage.service.StorageService
+
+/**
+ * The DevToolsStarter class can be used to start the [DevToolsService]. Requires the [Context]
+ * to start the [DevToolsService].
+ */
+class DevToolsStarter(private val context: Context) {
+
+  @VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
+  val devToolsIntent: Intent = Intent(context, DevToolsService::class.java)
+  private val bundle = Bundle()
+
+  /**
+   * Start the [DevToolsService] with the default values.
+   */
+  fun start() {
+    devToolsIntent.putExtras(bundle)
+    startForegroundService(context, devToolsIntent)
+  }
+
+  /**
+   * Start the [DevToolsService] with a custom [StorageService] class.
+   */
+  fun start(storageService: StorageService) {
```

Comment:
This requires that the caller instantiates the StorageService class.. that is not really a normal thing to do in Android. (The Android OS is what instantiates components like services/activities/etc.). I think it'd be best to just pass a class here.

```kotlin
fun start(storageServiceClass: Class<StorageService>)
```

This function and the other are basically identical too..  so I think they can either:

1. be combined into one method with a default value for the storage service class, or
2. be renamed to `start()` and `startForStorageService(storageServiceClass)`

Alternatively, since the storage service class will not be changing at runtime (if this function is called by a client, it'll likely be called with the exact same arguments every time) - the storage service class could be made into a constructor argument.